libraries/AP_Frsky_Telem/AP_Frsky_Telem.h

```diff
@@ -112,7 +112,9 @@ for FrSky SPort Passthrough
 #define ATTIANDRNG_PITCH_OFFSET     11
 #define ATTIANDRNG_RNGFND_OFFSET    21
 // for fair scheduler
-#define TIME_SLOT_MAX               11
+#define TIME_SLOT_MAX               12
```

Comment:
I wonder if this could be derived somehow e.g. adding a `LAST` entry to passhtroughpackettype

Reply:
good idea, actually those #defines were not even used, my mistake.